app/templates/index.hbs

```diff
@@ -25,7 +25,7 @@
   {{#if this.model.callForSpeakersEvents}}
     <h2 class="main-heading">{{t 'Call for speakers'}}</h2>
     <div class="ui stackable three column grid">
-      {{#each this.cfsEvent as |event|}}
+      {{#each this.model.callForSpeakersEvents as |event|}}
```

Comment:
This shouldn't need a change now

Reply:
`{{#each this.model.callForSpeakersEvents as |event|}}` or `{{#each this.cfsEvent as |event|}}`

Reply:
The change

Reply:
I reverted it to previous didn't made any changes in this file.